sys-process/bashtop/bashtop-0.8.29.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Resource monitor that shows usage and stats"
+HOMEPAGE="https://github.com/aristocratos/bashtop"
+SRC_URI="https://github.com/aristocratos/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="curl lm-sensors sysstat"
+DOCS=( README.md )
+
+DEPEND="${RDEPEND}"
+RDEPEND="curl? ( >=net-misc/curl-7.16 )
+	lm-sensors? ( sys-apps/lm-sensors )
+	sysstat? ( app-admin/sysstat )
+	"
+src_prepare() {
+	eapply_user
```

Comment:
You should call `default` here instead, probably.

Reply:
I had to make changes to path of binary according to gentoo policy and also, remove redundant doc from the package.

Reply:
I'm not saying remove your definition of `src_prepare`, I just mean call `default` instead. See [here](https://devmanual.gentoo.org/ebuild-writing/functions/src_prepare/index.html) for why.

Reply:
Okay, got it.